### ISSUE TITLE
Issues in mobile Safari when used as Bootstrap input group

### ISSUE THREAD

**Issue by jackocnr** [Bluefieldscom/intl-tel-input#165]
Previous discussion in this issue: https://github.com/Bluefieldscom/intl-tel-input/issues/148
And this pull request: https://github.com/Bluefieldscom/intl-tel-input/pull/161
Issue can be reproduced in [this Codepen](http://codepen.io/jackocnr/pen/pvdgrO) by uncommenting the JS line relating to the arrow div.

Issue: cannot open dropdown or even focus the input in mobile safari when the input is part of a Bootstrap input group. Problem seems to be with the arrow div - removing it after initialising the plugin fixes the problem. Needs further investigation, but I'm assuming it's a problem with the CSS triangle.

**Comment by gegere** [comment]
I did like the arrow. RIP arrow. Good work.

**Comment by jackocnr** [comment]
Update: it is the CSS triangle. Add this line of CSS to remove the triangle and fix the problem:

    .intl-tel-input .iti-flag .arrow {border: none;}

I will continue to investigate and maybe file a bug with Bootstrap/Mobile Safari in due course.

**Comment by jackocnr** [comment]
I managed to whittle the test case down to this: http://codepen.io/jackocnr/pen/LEQNXx

And posted an issue with the Bootstrap guys here: https://github.com/twbs/bootstrap/issues/15842

I'll see what they say. The next job would be to extract the relevant CSS lines from the Bootstrap stylesheet and just post it on Stack Overflow and/or the Mobile Safari bug tracker, if one exists.

**Comment by jackocnr** [comment]
The Bootstrap guys confirmed the problem, but said it is a Webkit issue. So I managed to extract the 2 lines from Bootstrap required to reproduce the issue.

Codepen here: http://codepen.io/jackocnr/pen/LEdrxP

And posted an issue on the WebKit Bugzilla here: https://bugs.webkit.org/show_bug.cgi?id=141822 and also on bugreport.apple.com.

We'll see what they say. I also added a note to the readme about the workaround.
